src/PDBOutput.h

```diff
@@ -72,6 +72,9 @@ struct PDBOutput : OutputableBase {
   void InsertAtomInLine(std::string & line, XYZ const& coor,
                         std::string const& occ, double const& beta);
 
+  void InsertAtomInLine(std::string & line, XYZ const& coor,
+                        double const& occ, double const& beta);
+
```

Comment:
What's the difference between these two functions?

Reply:
Converted to template.